ui/components/ui/tabs/tabs.tsx

```diff
@@ -86,26 +86,31 @@ export const Tabs = <TKey extends string = string>({
   };
 
   return (
-    <Box className="tabs" {...props}>
+    <Box
+      className={twMerge('tabs', 'transform-gpu', props.className)}
+      {...props}
+    >
       <Box
-        as="ul"
-        display={Display.Flex}
-        justifyContent={JustifyContent.flexStart}
-        backgroundColor={BackgroundColor.backgroundDefault}
+        role="tablist"
+        flexDirection={BoxFlexDirection.Row}
+        justifyContent={BoxJustifyContent.Start}
+        backgroundColor={BoxBackgroundColor.BackgroundDefault}
         gap={0}
         {...tabListProps}
-        className={classnames(
+        className={twMerge(
           'tabs__list',
+          'top-0 z-[2]',
           tabsClassName,
-          tabListProps.className || '',
+          tabListProps?.className,
         )}
       >
         {renderTabs()}
       </Box>
       {subHeader}
       <Box
+        role="tabpanel"
         {...tabContentProps}
-        className={classnames('tabs__content', tabContentProps.className || '')}
+        className={twMerge('tabs__content', tabContentProps?.className || '')}
```

Comment:
Full migration of legacy props to new MMDS component api and tailwind classnames instead of SCSS which reduces CSS